MatrixSDK/Utils/MXHTTPClient.m

```diff
@@ -81,6 +86,17 @@ -(id)initWithBaseURL:(NSString *)baseURL andAccessToken:(NSString *)access_token
         
         httpManager = [[AFHTTPRequestOperationManager alloc] initWithBaseURL:[NSURL URLWithString:baseURL]];
         
+        // Check whether some certificates have been included in app bundle
+        AFSecurityPolicy *securityPolicy = [AFSecurityPolicy policyWithPinningMode:AFSSLPinningModeCertificate];
+        if (securityPolicy.pinnedCertificates.count)
+        {
+            securityPolicy.allowInvalidCertificates = YES;
```

Comment:
Is this option needed for AFNetworking to go & defer the cert validation to us via the callback? It seems like if we're doing the cert validation ourselves (which it looks like we are below) this should be redundant.